src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -37,15 +37,20 @@
 
 
 class TornadoResponse(object):
-    """This class registers tornadoes with arguments in the order they are called
+    """This class registers tornado methods with arguments in the order they are called
     from TornadoResponse to call them later.
 
-    Use::
+    This is used in exceptional cases, in most cases it is not required,
```

Comment:
https://www.tornadoweb.org/en/stable/web.html#thread-safety-notes
Tornado say that:
> In general, methods on RequestHandler and elsewhere in Tornado are not thread-safe. In particular, methods such as write(), finish(), and flush() must only be called from the main thread.

This is not the case with `TornadoService`, but handlers that inherit `TornadoREST` may be flexible enough to use **methods on RequestHandler**. So if you are afraid to use Tornado methods due to the warning above, you can use `TornadoResponse`, it will call your actions in the main thread.